kw/skycrapper/settings.py

```diff
@@ -1,7 +1,10 @@
 STALL_MAPPING = {
-    "FAIRY": 0,
-    "AIRWICK": 1,
-    "NESCAFE": 2,
-    "ARIEL": 3,
-    "FINISH": 4  # JUST FINISH ALREADY
+    "HARD_LUMPS": 0,
```

Comment:
zero should be like "nothing" no? when nobody used the stall

Reply:
`None` should be nothing, `0` is the first thing :)

Reply:
`None` is not a number. I bet any mathematician caring about shit (this time literally) would kill you for this suggestion.

Reply:
Lucky us, we are not making maths: https://www.cs.utexas.edu/users/EWD/transcriptions/EWD08xx/EWD831.html